src/binders/payments/parameters.ts

```diff
@@ -77,6 +77,13 @@ export type CreateParameters = Pick<
      * @see https://docs.mollie.com/reference/v2/payments-api/create-payment?path=cardToken#credit-card
      */
     cardToken?: string;
+    /**
+     * The unique identifier used for referring to a terminal. This ID is used for assigning the payment to a specific terminal and it can be retrieved via List terminals. For more information about
+     * point-of-sale payments, please check our guide point-of-sale payments.
+     *
+     * @see https://docs.mollie.com/reference/v2/payments-api/create-payment?path=terminalId#point-of-sale
+     */
+    terminalId?: string
```

Comment:
I cannot find this parameter in [create-payment](https://docs.mollie.com/reference/create-payment).

Is this a doc issue or does it not exist (anymore)?

Reply:
It's defined here: https://docs.mollie.com/reference/extra-payment-parameters#point-of-sale, we moved all the method-specific parameters to that section. Typing it as string is fine, we might want to update the `@see` link.

Reply:
this might be a misunderstanding.
Within the package 'parameter' refers to request parameters, that can be sent to the API.
'data' refers to response data, that is returned by the API.

I believe the link above refers to response data. specifically `payment.details`.
In fact there is a separate comment below, regarding this.

Here I'm specifically asking, if `terminalId` is allowed as a request parameter for `payment.create`, since it does not appear in the [docs](https://docs.mollie.com/reference/create-payment).

Reply:
@fjbender still need your clarification on this one, pls. :)

Reply:
I just realised that you actually linked to additional create params. so terminalId IS in there.

Thanks for the clarification.